bridge/client/index.html

```diff
@@ -7,19 +7,20 @@
   <script>
     // NOTE: if changed, update tests in app.component.spec.ts
     function getBridgeBaseHref(origin, path) {
-      if(path.indexOf('/bridge') != -1)
+      if (path.indexOf('/bridge') !== -1)
         return [origin, path.substring(0, path.indexOf('/bridge')), '/bridge/'].join('');
       else
         return origin;
     }
+
     document.getElementById("base-href").href = getBridgeBaseHref(window.location.origin, window.location.pathname);
   </script>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link id="appFavicon" rel="icon" type="image/png" href="assets/branding/logo_inverted.png">
+  <link id="appFavicon" rel="icon" type="image/png" href="assets/default-branding/logo_inverted.png">
```

Comment:
revert it to `assets/branding/logo_inverted.png`. The branding folder is created by the bridge-server on startup and is used for custom-look-and-feel.

Reply:
Ok didn't know that :)